src/main/kotlin/com/avail/interpreter/levelTwo/operation/L2_VIRTUAL_CREATE_LABEL.kt

```diff
@@ -0,0 +1,324 @@
+ *     list of conditions and the following disclaimer in the documentation
+ *     and/or other materials provided with the distribution.
+ *
+ *  * Neither the name of the copyright holder nor the names of the contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.avail.interpreter.levelTwo.operation
+
+import com.avail.descriptor.representation.NilDescriptor
+import com.avail.descriptor.types.ContinuationTypeDescriptor
+import com.avail.descriptor.types.FunctionTypeDescriptor
+import com.avail.descriptor.types.IntegerRangeTypeDescriptor
+import com.avail.descriptor.types.TypeDescriptor
+import com.avail.interpreter.levelTwo.L2Chunk.ChunkEntryPoint
+import com.avail.interpreter.levelTwo.L2Instruction
+import com.avail.interpreter.levelTwo.L2OperandType
+import com.avail.interpreter.levelTwo.L2Operation
+import com.avail.interpreter.levelTwo.operand.*
+import com.avail.interpreter.levelTwo.operand.TypeRestriction.RestrictionFlagEncoding
+import com.avail.interpreter.levelTwo.operation.L2_REIFY.StatisticCategory
+import com.avail.optimizer.L2ControlFlowGraph
+import com.avail.optimizer.L2ControlFlowGraph.ZoneType
+import com.avail.optimizer.L2Generator
+import com.avail.optimizer.jvm.JVMTranslator
+import com.avail.utility.PrefixSharingList
+import org.objectweb.asm.MethodVisitor
+import java.util.*
+import java.util.function.Consumer
+
+/**
+ * This is a placeholder instruction, which is replaced if still live after data
+ * flow optimizations by:
+ *
+ *  1. ensuring the caller is reified.  If we're not already in a reification
+ * [L2ControlFlowGraph.Zone], generate the following, with a dynamic
+ * [L2_JUMP_IF_ALREADY_REIFIED] check to skip past it:
+ *
+ *  1. [L2_REIFY],
+ *  1. [L2_ENTER_L2_CHUNK] (start of reification area),
+ *  1. [L2_SAVE_ALL_AND_PC_TO_INT], falling through to
+ *  1. [L2_GET_CURRENT_CONTINUATION],
+ *  1. [L2_GET_CURRENT_FUNCTION],
+ *  1. [L2_CREATE_CONTINUATION],
+ *  1. [L2_SET_CONTINUATION],
+ *  1. [L2_RETURN_FROM_REIFICATION_HANDLER], then outside the reification area,
+ *  1. [L2_ENTER_L2_CHUNK].
```

Comment:
I believe these require a deeper indent to ensure they're presented as a sublist as in the original Java.

Reply:
Actually, it still provides a sublist, however we can't indent them because we break the links of everything in []. Even indenting them 1 space doesn't fix the issue. I changed it to be bulleted and numbered. I put an image in slack